src/bin/main.rs

```diff
@@ -178,14 +198,20 @@ fn main() {
     }) {
         Some(inputs) => inputs,
         None => {
-            eprintln!(
-                "No inputs provided via arguments or toml file. For help, use:\n\
+            return Err(format!(
+                "ERROR: No inputs provided via arguments or toml file. For help, use:\n\
                  > yarner -h",
-            );
-            return;
+            ));
         }
     };
 
+    if inputs.is_empty() {
+        return Err(format!(
+            "ERROR: No input files found. For help, use:\n\
```

Comment:
@adamreichold Any idea why this does not produce a newline, neither in the console, nor when redirecting to a file?